node/tests/ServerModules.test.ts

```diff
@@ -280,3 +279,190 @@ describe("GlideJson", () => {
         },
     );
 });
+
+describe("GlideFt", () => {
+    let cluster: ValkeyCluster;
```

Comment:
I think you can reuse this instead of copy-pasting. Either move common code to a helper function or merge top-level `describe` blocks (you can define nested blocks inside).